docs/guide/url_management.md

```diff
@@ -1,36 +1,39 @@
 # URL management
 
-## External URL validation
+Manage external URL addresses and URL wildcards with a comfortable user interface that is available in the Back Office, Admin tab, the URL Manager node. Configure URL aliases to have human-readable URL addresses throughout your system.
+
+## Link manager
+
+When developing a site, users can enter links to external websites in either RichText or URL fields.  Each such link is then stored in the URL table. You can view and update all external links that exist within the site, without having to modify and re-publish the individual Content items.
+
+The Link manager tab contains all the information about each link, including its status (valid or invalid) and the time the system last attempted to validate the URL address. Click an entry in the list to display its details and check which Content items use this link. Edit the entry to update the URL address in all the occurrences throughout the website.
+
+!!! note
 
-Every link that is input into a RichText or URL Field is stored in the URL table.
-You can view and edit published URLs in the Link manager without having to modify the Content items.
-This means that you don't have to edit and re-publish your content if you just want to change a link.
+    When you edit the details of an entry to update the URL address, the status automatically changes to valid.
 
-The Link manager contains all the necessary information about each address including its status (valid or invalid)
-and the time it was last checked (when the system attempted to validate the URL).
-By default, all URLs are valid.
 
-You can use the `ezplatform:check-urls` command to check all the addresses stored in the URL table.
-It validates the links by accessing them one by one.
-If a broken link is found, its status is set to "invalid". The last checked field is always updated.
+## External URL validation
+
+You can manually validate all the addresses from the URL table, by executing the `ezplatform:check-urls` command. It validates the links by accessing them one by one and updates the value in the Last checked field. If a broken link is found, its status is set to "invalid".
```

Comment:
```suggestion
You can validate all the addresses from the URL table by executing the `ezplatform:check-urls` command. It validates the links by accessing them one by one and updates the value in the Last checked field. If a broken link is found, its status is set to "invalid".
```
The same command can be run automatically, so I'd get rid ot the "manually"

Reply:
Automatic validation is mentioned below, hence the word "manually" to underline the difference between the two processes.